lib/assertions/fed.js

```diff
@@ -18,7 +18,7 @@ var assertKeyControl = (federates) => (begin, end, amountToTransferInWeis) => {
     var federateAddresses = federates
       .slice(begin, end)
       .map((federate) => rskClient.rsk.utils.publicKeyToAddress(
-          bitcoin.keys.publicKeyToUncompressed(federate.publicKeys.rsk)
+          bitcoin.keys.publicKeyToUncompressed(removePrefix0x(federate.publicKeys.rsk))
```

Comment:
Now we include the `0x` for the public keys in the config files. But this is expecting it to not have the `0x` prefix.